extensions/spark/kyuubi-spark-authz/src/main/resources/table_command_spec.json

```diff
@@ -157,7 +157,7 @@
 }, {
   "classname" : "org.apache.spark.sql.catalyst.plans.logical.CreateTableAsSelect",
   "tableDescs" : [ {
-    "fieldName" : "left",
+    "fieldName" : "name",
```

Comment:
There is no `left` method in spark 3.5.

https://github.com/apache/spark/blob/e056fc8fb4d1c1ab8786a0989adc7298cc918dae/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala#L444

https://github.com/apache/spark/blob/64e2b22f6b4023197871a60eb08b055688e9fdd2/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala#L467